app/assets/stylesheets/hackney_new_repair_request.scss

```diff
@@ -0,0 +1,3 @@
+.hackney-repair-request-description-hint {
+  margin-top: -20px;
```

Comment:
Do we really need that margin now? And can't it be done with `govuk-` margin classes?

Reply:
I'll use the govuk- margin class, always forget about that! I think we do need it, if we're following the general gds layout for this kind of thing

Reply:
Ohh so you copied this from the latest version?

Reply:
<img width="801" alt="Screenshot 2019-06-19 at 10 56 27" src="https://user-images.githubusercontent.com/35098639/59756063-ed869480-9280-11e9-9bd9-418f45a18173.png">
I'm following this design